components/dataentry/new-event-controller.js

```diff
@@ -298,13 +300,29 @@ trackerCapture.controller('EventCreationController',
                         }
                         $scope.orgUnitsLoading = false;
                     });
-                }
-
-                
             });
             
         },350);
     }
+
```

Comment:
Can these two functions getOrgUnit and getOrgUnits be removed again? We ended up using the existing functions from the central services.js file, right?